src/main/java/com/google/api/codegen/metacode/FieldStructureParser.java

```diff
@@ -76,7 +76,10 @@ private static InitValueConfig createInitValueConfig(
               .get(fieldConfig.fieldPath())
               .withInitialCollectionValue(fieldConfig.entityName(), fieldConfig.value());
     } else if (fieldConfig.hasSimpleInitValue()) {
-      valueConfig = InitValueConfig.createWithValue(stripQuotes(fieldConfig.value()));
+      InitValue initValue = fieldConfig.value();
+      valueConfig =
+          InitValueConfig.createWithValue(
+              InitValue.create(stripQuotes(initValue.getValue()), initValue.getType()));
```

Comment:
The quote stripping is used at multiple places in FieldStructureParser so I would prefer to keep it to this class.